docs/04-dacs.md

```diff
@@ -0,0 +1,69 @@
+# DAC Configuration
+
+While most Raspberry Pi boards come with an onboard sound card it is well known that the quality you can get from them is not the greatest. The Pi Foundation makes an excelent job at keeping the cost of their bords down, but that comes with some compromises, audio being one of them: the [audio circuitry](https://hackaday.com/2018/07/13/behind-the-pin-how-the-raspberry-pi-gets-its-audio/) does an OK job but it's nothing stellar.
+
+If you want to upgrade the sound quality of your balenaSound devices (or if you are using a board that does not have onboard sound card such as the Raspberry Pi Zero) you will need to add a DAC board (Digital Audio Converter) to your project.
```

Comment:
If you want to upgrade the sound quality of your balenaSound devices (or if you are using a board that does not have onboard sound card such as the Raspberry Pi Zero), you'll need to add a DAC board (Digital Audio Converter) to your project.